packages/clay-css/src/scss/mixins/_cards.scss

```diff
@@ -277,146 +412,505 @@
 
 		.aspect-ratio {
-			@include clay-aspect-ratio-variant($aspect-ratio);
+			@include clay-aspect-ratio-variant(
+				setter(map-get($map, aspect-ratio), ())
+			);
+
+			@if ($aspect-ratio-checkered-fg) {
+				@include clay-bg-checkered($aspect-ratio-checkered-fg);
+			}
+
+			.custom-control label,
+			.form-check-label {
+				@include clay-css(
+					setter(
+						map-deep-get($aspect-ratio, custom-control, label),
+						()
+					)
+				);
+			}
+
+			.label {
+				@include clay-label-variant(
+					setter(map-get($aspect-ratio, label), ())
+				);
+			}
 		}
 
 		.aspect-ratio-item {
-			@include clay-aspect-ratio-item-variant($aspect-ratio-item);
+			@include clay-aspect-ratio-item-variant(
+				setter(map-get($map, aspect-ratio-item), ())
+			);
+		}
+
+		.aspect-ratio-item-top-left {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-top-left), ())
+			);
+		}
+
+		.aspect-ratio-item-top-center {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-top-center), ())
+			);
+		}
+
+		.aspect-ratio-item-top-right {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-top-right), ())
+			);
+		}
+
+		.aspect-ratio-item-right-middle {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-right-middle), ())
+			);
+		}
+
+		.aspect-ratio-item-bottom-right {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-bottom-right), ())
+			);
+		}
+
+		.aspect-ratio-item-bottom-center {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-bottom-center), ())
+			);
+		}
+
+		.aspect-ratio-item-bottom-left {
+			@include clay-css(
+				setter(map-get($map, aspect-ratio-item-bottom-left), ())
+			);
 		}
 
 		.sticker {
-			@include clay-sticker-variant($sticker);
+			@include clay-sticker-variant(setter(map-get($map, sticker), ()));
+		}
+
+		.card {
+			@include clay-css(setter(map-get($map, card), ()));
 		}
 
 		.card-header {
-			@include clay-card-section-variant($card-header);
+			@include clay-card-section-variant(
+				setter(map-get($map, card-header), ())
+			);
 		}
 
 		.card-body {
 			@include clay-card-section-variant($card-body);
 		}
 
 		.card-footer {
-			@include clay-card-section-variant($card-footer);
+			@include clay-card-section-variant(
+				setter(map-get($map, card-footer), ())
+			);
 		}
 
 		.card-row {
 			@include clay-card-section-variant($card-row);
 		}
 
 		.card-title {
-			@include clay-link($card-title);
+			@include clay-link(setter(map-get($map, card-title), ()));
 		}
 
 		.card-subtitle {
-			@include clay-link($card-subtitle);
+			@include clay-link(setter(map-get($map, card-subtitle), ()));
 		}
 
 		.card-text {
-			@include clay-link($card-text);
+			@include clay-link(setter(map-get($map, card-text), ()));
 		}
 
 		.card-link {
-			@include clay-link($card-link);
+			@include clay-link(setter(map-get($map, card-link), ()));
+		}
+
+		.card-divider {
+			@include clay-css(setter(map-get($map, card-divider), ()));
+		}
```

Comment:
Just one question I'm not sure if I already asked about it, when we don't have the `card-divider` in `$map` the `.card-divider` won't come out in the final build?

Reply:
Yeah `clay-css` won't output anything, it would be equivalent of writing:
```scss
.card .card-divider {}
```
Sass doesn't output empty rule-sets.

Reply:
Oh thanks for clarifying Patrick.